x-pack/plugins/security_solution/common/search_strategy/security_solution/risk_score/common/index.ts

```diff
@@ -24,7 +24,9 @@ export const getHostRiskIndex = (
   isNewRiskScoreModuleInstalled: boolean
 ): string => {
   return isNewRiskScoreModuleInstalled
-    ? getRiskScoreLatestIndex(spaceId)
```

Comment:
I found this bug that we didn't consider the `onlyLatest` property for the host index. I fixed it by copying the user index code.

Reply:
https://github.com/elastic/kibana/blob/64ea66731bee85806f28e19de881deaa2e2e5842/STYLEGUIDE.mdx#L401-L414